app/services/queue_links_for_publishing.rb

```diff
@@ -1,4 +1,4 @@
-class PublishTags
+class QueueLinksForPublishing
```

Comment:
As discussed IRL, aiming for consistency in the verb vs noun naming of these service objects.